draft-ietf-quic-recovery.md

```diff
@@ -284,12 +284,12 @@ frame.  For more information, see Section 13.2 of {{QUIC-TRANSPORT}}.
 ## Computing the RTT estimate
 
 RTT is calculated when an ACK frame arrives by computing the difference between
-the current time and the time the largest newly acked packet was sent.  If no
-packets are newly acknowledged, RTT cannot be calculated. When RTT is
-calculated, the ack delay field from the ACK frame SHOULD be subtracted from the
-RTT as long as the result is larger than the Min RTT.  If the result is smaller
-than the min_rtt, the RTT should be used, but the ack delay field should be
-ignored.
+the current time and the time the largest acked packet was sent.  If the largest
+acked is not newly acknowledged or not retransmittable, RTT cannot be
+calculated. When RTT is calculated, the ack delay field from the ACK frame
```

Comment:
I would make this normative. "An RTT sample MUST NOT be taken for a packet that is not newly acknowledged or not retransmittable."

Reply:
Thanks, updated